lib/pengine/bundle.c

```diff
@@ -180,16 +178,17 @@ create_docker_resource(
         char *dbuffer = calloc(1, dmax+1);
 
         char *id = NULL;
-        xmlNode *xml_docker = NULL;
+        xmlNode *xml_container = NULL;
         xmlNode *xml_obj = NULL;
 
-        id = crm_strdup_printf("%s-docker-%d", data->prefix, tuple->offset);
+        id = crm_strdup_printf("%s-docker-%d", data->prefix, replica->offset);
         crm_xml_sanitize_id(id);
-        xml_docker = create_resource(id, "heartbeat", "docker");
+        xml_container = create_resource(id, "heartbeat", "docker");
```

Comment:
Probably makes more sense to reference PE_*_DOCKER_S here.

Reply:
Good call